andela/hc-logros-infinitos
# Ft nag user 153728008 - Nag user notifications

## What does this PR do?*

• Add the nag user functionality.

## Description of task to be completed

• The system should keep nagging the user every so often until they resolve the job that has gone down.

## How should this be manually tested

- Add a check. 
- Add the period, grace and nag-interval times
- Ping it the first time to be up.
- Wait for the period and grace to expire. Email will be sent for down notification.
- Wait for nag period to expire. Email will be sent to nag.
- This will happen again and again.

## Any background context you want to provide

• When a User's job goes down, the user is sent a notification email informing him of the same. We added the nag functionality for more notifications.

## What are the relevant pivotal Tracker stories*

• [#153728008](https://www.pivotaltracker.com/story/show/153728008)
